azure/scope/machine.go

```diff
@@ -128,13 +128,12 @@ func (m *MachineScope) InitMachineCache(ctx context.Context) error {
 
 		m.cache.VMSKU, err = skuCache.Get(ctx, m.AzureMachine.Spec.VMSize, resourceskus.VirtualMachines)
 		if err != nil {
-			return azure.WithTerminalError(errors.Wrapf(err, "failed to get VM SKU %s in compute api", m.AzureMachine.Spec.VMSize))
+			return err
```

Comment:
was it intentional to remove the wrapped error?
```suggestion
			return errors.Wrapf(err, "failed to get VM SKU %s in compute api", m.AzureMachine.Spec.VMSize)
```

Reply:
I think I removed it thinking the wrapped error message was redundant at the time but looking back on it it's probably better to leave it as you suggest. Added back.